source/administration/config-settings.rst

```diff
@@ -4863,6 +4863,43 @@ Changes to this setting require a server restart before taking effect.
 | This feature's ``config.json`` setting is ``"DataSourceSearchReplicas": []`` with string array input consisting of database connection strings.   |
 +---------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Replica Lag Settings
+^^^^^^^^^^^^^^^^^^^^
+
+*Available in Enterprise Edition E20*
+
+Specifies a connection string and user-defined SQL queries on the database to measure replica lag for a single replica instance. These settings monitor absolute lag based on binlog distance/transaction queue length, and the time taken for the replica to catch up.
+
++-------------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"ReplicaLagSettings": []`` with string array input.       |
++-------------------------------------------------------------------------------------------------------+
+
+String array input can consist of:
```

Comment:
Perhaps the word "can" implies that other input can also be there? Maybe it's good to just say: "String array input consists of:"